packages/core/src/utils/TypeUtil.ts

```diff
@@ -18,6 +18,30 @@ import type { ConstantsUtil } from './ConstantsUtil.js'
 import type { ReownName } from '../controllers/EnsController.js'
 import type UniversalProvider from '@walletconnect/universal-provider'
 
+type AppKitConfigs = {
```

Comment:
Can we give another name specific to event to not confuse with other type definitions? Like `InitializeEventAppKitParams`